Lib/test/test_binascii.py

```diff
@@ -114,6 +114,38 @@ def addnoise(line):
         # empty strings. TBD: shouldn't it raise an exception instead ?
         self.assertEqual(binascii.a2b_base64(self.type2test(fillers)), b'')
 
+    def test_base64_strict_mode(self):
+        # Test base64 with strict mode on
```

Comment:
in all of these strict mode assertion tests, it'd be good to also validate the behavior with an explicit strict_mode=False and no strict_mode= parameter supplied at all (the default).  So that we're testing all scenarios.

That basically just means calling `binascii.a2b_base64(self.type2test(data))` outside of an assertRaises block, potentially also asserting the value it decoded to (that'd be an additional parameter).

That way we gain an explicit test for what the permissive behavior is as well.

Reply:
Done in 5f8df5b.